code/game/objects/structures/watercloset.dm

```diff
@@ -270,18 +270,23 @@
 		mobpresent--
 	..()
 
-//Yes, showers are super powerful as far as washing goes.
+//Yes, showers are super powerful as far as washing goes
+//Shower cleaning has been nerfed (no, really). 75 % chance to clean everything on each tick
+//You'll have to stay under it for a bit to clean every last noggin
+
+#define CLEAN_PROB 75 //Percentage
+
 /obj/machinery/shower/proc/wash(atom/movable/O as obj|mob)
 	if(!on)
 		return
 
 	if(iscarbon(O))
```

Comment:
That would quite change the way it works though. The idea is every part of your body has a 75 % chance to clean each tick

I guess I could make it prettier by making a list of everything to be cleaned, putting it in a for loop and running the prob(75) here